base/index.scss

```diff
@@ -1,3 +1,4 @@
+@import '../bower_components/flexboxgrid/dist/flexboxgrid';
```

Comment:
i would move this import to `dist/styles.scss`, because `index` should contain partials in this directory not from outside